portfolio/src/main/webapp/comments.js

```diff
@@ -12,16 +12,81 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function loadComments(){
-  const maxComments = parseInt(document.getElementById("max-comments-select").value);
-  debugLog(maxComments);
-  fetch("/data" + "?max_comments=" + maxComments).then(response => response.json()).then((comments) => {
-    debugLog(comments);
-    const commentsList = document.getElementById('comments-container');
-    commentsList.innerHTML = '';
-    comments.forEach((comment) => {
-      commentsList.appendChild(createCommentElement(comment.email, comment.comment, comment.date));
-    });
+function loadPage(pgNumber = 1){
+  const commentsPerPage = parseInt(document.getElementById('comments-per-page-select').value);
+  
+  debugLog("commentsPerPage=" + commentsPerPage);
+  fetch("/data" + "?comments_per_page=" + commentsPerPage+ "&pg_number=" + pgNumber).then(response => response.json()).then((response) => {
+    loadComments(response.comments);
+    loadPagination(response.lastPage, pgNumber);
+  });
+  setTimeout(function(){
+    addClass(document.getElementById("pg" + pgNumber), "active");
```

Comment:
What is this doing? It appears to set an "active" class on a particular page element after a delay. Consider adding a comment above it?